internal/collector/settings.go

```diff
@@ -73,6 +73,26 @@ func createURIs(cfg *config.Config) []string {
 	return []string{cfg.Collector.ConfigPath}
 }
 
+func createFile(err error, confPath string) error {
```

Comment:
Can you remove the error from the function parameters and perform that check before calling this function?